edge-agent/docker/linux/amd64/Dockerfile

```diff
@@ -9,8 +9,9 @@ ARG EXE_DIR=.
 RUN apk update && \
     apk add --no-cache snappy
 
-# create a user to allow agent to optionally run as non-root
-ARG EDGEAGENTUSER_ID=1000	
+# create a default user to allow agent to run as non-root
+ARG EDGEAGENTUSER_ID
+ENV EDGEAGENTUSER_ID ${EDGEAGENTUSER_ID:-1000}
 RUN adduser -Ds /bin/sh -u ${EDGEAGENTUSER_ID} edgeagentuser 
```

Comment:
Is there a point in still doing this since the entrypoint will do it anyway?

Reply:
I have the same comment given the script setup the environment variable & default value as well.

Reply:
This provides the user a UID they can set to run as non-root at start.  This is also needed for Edge on K8s, as some clusters we support have a policy to start running as non-root.

Reply:
Ah yes, forgot about that.